mongodb-advanced/components/person-editor-component.html

```diff
@@ -0,0 +1,38 @@
+<br-modal br-title="Add a Person to the list">
+  <br-modal-body>
+    <div layout-padding>
+      <div layout="row">
+        <label>
+          Enter a name:
+          <input type="text" ng-model="$ctrl.person.name" ex-focus="true" />
```

Comment:
I think it's good not to introduce `br-input` (and bedrock-angular-form) into basic examples, however, I think it would be appropriate to use `mdInput` here in keeping with the material theme.  Since there is already the ngMaterial dependency, might as well take advantage of it.